.github/workflows/checks.yml

```diff
@@ -117,12 +144,12 @@ jobs:
           sudo sysctl -w kernel.apparmor_restrict_unprivileged_userns=0
 
       - name: Setup Python & Poetry Environment
-        uses: exasol/python-toolbox/.github/actions/python-environment@0.14.0
+        uses: exasol/python-toolbox/.github/actions/python-environment@0.18.0
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Calculate Test Coverage
-        run: poetry run nox -s coverage
+        run: poetry run nox -s test:coverage
```

Comment:
Maybe we should move the integration tests to the slow tests, too?
I.e. not run the (also quote slow) integration tests here.
See how to override the nox task `coverage` here  
https://github.com/exasol/bucketfs-python/blob/main/noxfile.py